tripy/tripy/backend/mlir/executor.py

```diff
@@ -126,12 +128,11 @@ def _get_output_tensor_info(self, outputs_runtime_shape, output_devices):
         return outputs_tensor_info
 
     def get_output_tensor_runtime_info(self, inputs, output_devices=List[device]):
-        inputs_shape_memref = self._get_inputs_shape_memref(
-            inputs
-        )  # Can we use executable signature inputs to retrieve this information?
```

Comment:
A: We can't, the information in the signature is only for compilation, but we need runtime information here.